java/src/jmri/jmrix/lenz/XNetReply.java

```diff
@@ -686,6 +810,12 @@ public boolean isUnsolicited() {
                 || (this.isFeedbackMessage() && reallyUnsolicited));
     }
 
+    /**
+     * Resets the unsolicited feedback flag. If the feedback was received
```

Comment:
Why’ll feedback messages may be the primary user of this, this doesn’t just apply to feedback messages.